pom.xml

```diff
@@ -32,8 +32,8 @@
         <maven.deploy.skip>true</maven.deploy.skip>
         <maven.site.skip>true</maven.site.skip>
         <graylog.version>3.2.5-SNAPSHOT</graylog.version>
-        <aws-java-sdk-2.version>2.10.41</aws-java-sdk-2.version>
-        <aws-kinesis-client.version>2.2.6</aws-kinesis-client.version>
+        <aws-java-sdk-2.version>2.13.6</aws-java-sdk-2.version>
```

Comment:
There seems to be a newer version `2.13.8` https://github.com/aws/aws-sdk-java-v2